lib/fastlane/plugin/aws_device_farm/actions/aws_device_farm_action.rb

```diff
@@ -68,7 +68,9 @@ def self.run(params)
         # rubocop:disable  Metrics/BlockNesting
         if params[:wait_for_completion]
           UI.message 'Waiting for the run to complete. ☕️'
-          run = wait_for_run project, run, params
+          run = wait_for_run project, run, params[:print_waiting_periods]
```

Comment:
i'd prefer to keep sending in the whole `params` hash

Reply:
ok. i set params.